modules/installing-gitops-operator-in-web-console.adoc

```diff
@@ -10,10 +10,45 @@ You can install {gitops-title} Operator from the OperatorHub by using the web co
 
 .Procedure
 
-. Open the *Administrator* perspective of the web console and navigate to *Operators* → *OperatorHub* in the menu on the left.
+. Open the *Administrator* perspective of the web console and go to *Operators* -> *OperatorHub*.
 
 . Search for `OpenShift GitOps`, click the *{gitops-title}* tile, and then click *Install*.
+
+. On the *Install Operator* page:
+
+.. Select an *Update channel*.
+
+.. Select a {gitops-shortname} *Version* to install.
+
+.. Choose an *Installed Namespace*. The default installation namespace is `openshift-gitops-operator`.
++
+[WARNING]
+====
+For the {gitops-shortname} version 1.10 and later, the default namespace changed from `openshift-gitops` to `openshift-gitops operator`. If the Operator is installed in a different namespace and undergoes an upgrade from version 1.9 to version 1.10, it can result in unexpected behavior.
+====
```

Comment:
@Srivaralakshmi I am not happy with this admonition being placed here as it is not specific  just to the installation in web console. But the introduction to the installation section already has two long admonitions so I do not think it is a good idea to add an another one. 

I wanted to hear your opinion on this. Maybe we could leave it there and also add it in the CLI section? But then we would have two same admonitions in two consecutive sections, which is also not ideal. :/